app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,10 @@
+# Dash-Board  Controller
```

Comment:
@sohilp393, Top-level comments should **describe** the Class instead of adding the class Name. 
cc: @karanvalecha